presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +metadata may be removed from server. If value is low,  +
++                                          +                                          +                                    +it's possible that client will not be able to receive  +
++                                          +                                          +                                    +information about query completion. The value          +
++                                          +                                          +                                    +describes minimum time that must pass to remove query  +
++                                          +                                          +                                    +(after it's considered completed) but if there is      +
++                                          +                                          +                                    +space available in history queue the query data will   +
++                                          +                                          +                                    +be kept longer. The size of history queue is defined   +
++                                          +                                          +                                    +by ``query.max-history`` property (``100`` by          +
++                                          +                                          +                                    +default).                                              +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++     ``query.max-concurrent-queries``     +       ``Integer`` (at least ``1``)       +              ``1000``              +**Deprecated** Describes how many queries be processed +
++                                          +                                          +                                    +simultaneously in single cluster node. It shouldn't be +
++                                          +                                          +                                    +used in new configuration, the                         +
++                                          +                                          +                                    +``query.queue-config-file`` can be used instead.       +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++           ``query.max-memory``           +          ``String`` (data size)          +             ``20 GB``              +Serves as default value for ``query_max_memory``       +
```

Comment:
I kinda thought I've did that, but probably it can be expressed more clearly.
To be done.